public/mockServiceWorker.js

```diff
@@ -2,13 +2,15 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (1.3.5).
+ * Mock Service Worker.
  * @see https://github.com/mswjs/msw
  * - Please do NOT modify this file.
  * - Please do NOT serve this file on production.
```

Comment:
Are we sure we're not serving this file in production?

Reply:
Not on this branch https://next.www.documentcloud.org/mockServiceWorker.js/

But some _other_ branches https://legacy.www.documentcloud.org/mockServiceWorker.js